WordPressAuthenticator/GoogleSignIn/Data+Base64URL.swift

```diff
@@ -0,0 +1,24 @@
+extension Data {
+
+    /// "base64url" is an encoding that is safe to use with URLs.
+    /// It is defined in RFC 4648, section 5.
+    ///
+    /// See:
+    /// - https://tools.ietf.org/html/rfc4648#section-5
+    /// - https://tools.ietf.org/html/rfc7515#appendix-C
+    init?(base64URLEncoded: String) {
```

Comment:
We need this type of decoding because of JWT. 

I considered making the `init` `throw`, but decided against because:

1. Existing `Data` initializers, such as `Data(base64Encoded:, options:)` are `init?`
2. I don't think we need a refined error granularity at this level.